packages/blockchain-api/app.alfajoresstaging.yaml

```diff
@@ -3,10 +3,11 @@ service: alfajoresstaging
 env_variables:
   NODE_ENV: 'development'
   DEPLOY_ENV: 'alfajoresstaging'
+  EXCHANGE_RATES_API: 'https://api.exchangeratesapi.io'
   BLOCKSCOUT_API: 'https://alfajoresstaging-blockscout.celo-testnet.org/api'
   # Pull addresses from the build artifacts of the network in protocol/build
   CELO_GOLD_ADDRESS: '0x1313e2f3EBef8f0d869EECEb796D55A066eEA863'
-  CELO_DOLLAR_ADDRESS: '0x2df4dd6bd1b26a8503f763506bdb8e7cf165f69e'
+  CELO_DOLLAR_ADDRESS: '0x2dF4dD6Bd1b26a8503F763506bdB8e7cf165f69E'
```

Comment:
is the uppercase needed?

Reply:
Not strictly, no. Just copied and pasted over the addresses from the artifacts again to be sure they were right